addons/sale_stock/views/stock_production_lot.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="stock_production_lot_sale_form_view" model="ir.ui.view">
```

Comment:
Please follow the naming guidelines: https://www.odoo.com/documentation/master/reference/guidelines.html

This should be named ``stock_production_lot_view_form``